keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -443,21 +444,32 @@ keeping the same behavior that exists today.
 #### API
 
 ```go
-// ServiceCIDR defines a range of IPs using CIDR format (192.168.0.0/24 or 2001:db2::0/64).
+// ServiceCIDR defines a range of IP addresses using CIDR format (e.g. 192.168.0.0/24 or 2001:db2::/64).
+// This range is used to allocate ClusterIPs to Service objects.
 type ServiceCIDR struct {
- metav1.TypeMeta   `json:",inline"`
- metav1.ObjectMeta `json:"metadata,omitempty"`
-
- Spec   ServiceCIDRSpec   `json:"spec,omitempty"`
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	// spec is the desired state of the ServiceCIDR.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+	// +optional
+	Spec ServiceCIDRSpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
+	// status represents the current state of the ServiceCIDR.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+	// +optional
+	Status ServiceCIDRStatus `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
 }
 
-
 // ServiceCIDRSpec describe how the ServiceCIDR's specification looks like.
 type ServiceCIDRSpec struct {
- // IPv4 is an IPv4 block in CIDR notation "10.0.0.0/8" 
- IPv4 string `json:"ipv4"`
- // IPv6 is an IPv6 block in CIDR notation "fd12:3456:789a:1::/64" 
- IPv6 string `json:"ipv6"`
+	// CIDRs defines the IP blocks in CIDR notation (e.g. "192.168.0.0/24" or "2001:db8::/64")
+	// from which to assign service cluster IPs. Max of two CIDRs is allowed, one of each IP family.
```

Comment:
hmm, good point, there is always a concept of primary IP family that is the one provided by the flags https://github.com/kubernetes/kubernetes/blob/164276175fb2fa36d223e3fdeae799d8cf86b0cd/pkg/registry/core/rest/storage_core.go#L354-L358 

We need this also because the kubernetes.default is single stack, despite the users can modify the service cidrs.

So, right now, this is as this:
- The API server will have a Primary IP family on dual stack cluster, that is provided via the order of the IP families in the Service CIDR flags
- This pimary IP family will be used as default for the creation of Services, special case is the kubernetes.default service
- The order of the CIDRs in the ServiceCIDRs object does not matter, although, the order of the CIDRs in the default ServiceCIDR represent the order of the flags in the apiserver, hence, the first family will be the default one